src/test/test_step.go

```diff
@@ -179,6 +179,11 @@ func test(tid int, state *core.BuildState, label core.BuildLabel, target *core.B
 		}
 	}
 
+	state.LogBuildResult(tid, target, core.TargetTesting, "Acquiring file lock...")
```

Comment:
`Acquiring target lock...` would be more descriptive I think.

Reply:
Agreed